booster/tools/booster/Proxy.hs

```diff
@@ -92,16 +90,16 @@ respondEither cfg@ProxyConfig{boosterState} booster kore req = case req of
                     LogSettings
                         { logSuccessfulRewrites = execReq.logSuccessfulRewrites
                         , logFailedRewrites = execReq.logFailedRewrites
-                        , logFallbacks = execReq.logFallbacks
-                        , logTiming = execReq.logTiming
+                        , logFallbacks = Nothing
+                        , logTiming = Nothing
```

Comment:
Can we get rid of this field or is there a reason to hold off?

Reply:
Oops, forgot to push!